src/test/java/seedu/guest/model/GuestBookTest.java

```diff
@@ -95,4 +96,10 @@ public ObservableList<Guest> getGuestList() {
         }
     }
 
+    @Test
+    public void hashcode() {
+        assertEquals(guestBook.hashCode(), new GuestBook().hashCode());
```

Comment:
Should we format this code block similar to other `hashcode` tests? i.e., line spacing with comments for each assertion.

Reply:
Agreed. Added line spacing and comments.